FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.SchemaManagement/ContainerRegistry/ContainerRegistryAccessTokenProvider.cs

```diff
@@ -89,6 +91,12 @@ public async Task<string> GetTokenAsync(string registryServer, CancellationToken
                 _logger.LogInformation(ex, "Failed to get ACR access token with AAD access token.");
                 throw new ContainerRegistryTokenException("Failed to get ACR access token with AAD access token.", ex);
             }
+            catch (ContainerRegistryTokenException ex)
+            {
+                _diagnosticLogger.LogError("Failed to get ACR access token with AAD access token.");
+                _logger.LogInformation(ex, "Failed to get ACR access token with AAD access token.");
+                throw;
```

Comment:
Do we need to catch ContainerRegistryTokenException again?

Reply:
Resolved it after offline discussion.
Catch here avoid falling into unhandled exception.